services/submission.js

```diff
@@ -1402,10 +1402,17 @@ class Submission {
                 [USER.ROLES.DATA_COMMONS_PERSONNEL],
                 aSubmission?.dataCommons
             ),
-            this.userService.findUsersByNotificationsAndRole(
-                [EN.DATA_SUBMISSION.CREATE],
-                [USER.ROLES.ADMIN, USER.ROLES.FEDERAL_LEAD]
```

Comment:
Shouldn't we filter DCP user by Data Commons also?

Reply:
This is not DCP role users. we are filtering DCP role above.

Reply:
GH didn't associate with correct lines. I meant code at 1400-1404